Addition in resources section

Rishit-dagli opened the pull request:
Added Coursera course Machine Learning with TensorFlow on GCP

Rishit-dagli committed:
Added in resources section

Added Coursera course Machine Learning with TensorFlow on GCP

Rishit-dagli commented:
@googlebot I signed it!

kyscg commented on README.md:
```diff
@@ -142,6 +142,7 @@ Build Type                                                        | Status
 *   [Getting Started with TensorFlow 2 from Coursera](https://www.coursera.org/learn/getting-started-with-tensor-flow2)
 *   [Intro to TensorFlow for Deep Learning from Udacity](https://www.udacity.com/course/intro-to-tensorflow-for-deep-learning--ud187)
 *   [Introduction to TensorFlow Lite from Udacity](https://www.udacity.com/course/intro-to-tensorflow-lite--ud190)
+*   [Machine Learning with TensorFLow on GCP](https://www.coursera.org/specializations/machine-learning-tensorflow-gcp)
```
```suggestion
*   [Machine Learning with TensorFlow on GCP](https://www.coursera.org/specializations/machine-learning-tensorflow-gcp)
```

Rishit-dagli replied:
Thanks for pointing that out, seems like a typo

kyscg reviewed (changes requested):
Good addition :+1:

Rishit-dagli committed:
Fixed a minor typo in resources section

Fixed Typo in the resources section from Machine Learning with TensorFLow on GCP to Machine Learning with TensorFlow on GCP

Co-authored-by: Kilaru Yasaswi Sri Chandra Gandhi <yasaswisrichandragandhi@gmail.com>

Rishit-dagli commented:
@kyscg Thank you for pointing out the typo in the resources section, I have now resolved it.
